editor/store/selectors.js

```diff
@@ -89,7 +89,7 @@ export function isEditedPostNew( state ) {
  * @return {boolean} Whether unsaved values exist.
  */
 export function isEditedPostDirty( state ) {
-	return state.editor.isDirty;
+	return state.editor.isDirty || inSomeHistory( state, isEditedPostDirty );
```

Comment:
This is causing a regression for the post publish panel, it's getting closed right after publish